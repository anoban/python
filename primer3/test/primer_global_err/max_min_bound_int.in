SEQUENCE_ID=max_min_bound_int
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_INTERNAL_MIN_BOUND=97.0
PRIMER_INTERNAL_OPT_BOUND=95.0
PRIMER_INTERNAL_MAX_BOUND=96.0
=
