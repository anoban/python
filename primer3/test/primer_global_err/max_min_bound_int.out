SEQUENCE_ID=max_min_bound_int
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_INTERNAL_MIN_BOUND=97.0
PRIMER_INTERNAL_OPT_BOUND=95.0
PRIMER_INTERNAL_MAX_BOUND=96.0
PRIMER_ERROR=Optimum internal oligo fraction binding lower than minimum or higher than maximum
=
